### PR TITLE
Nr manage fix

### PR DESCRIPTION
**Pull request by NR229**
What we were discussing during retro.
Updated the structure of seeds.rb and some external pages to integrate changes with manage pages feature.

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -8,12 +8,12 @@
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
 
-if Page.all.count < 3
-  Page.create([
-    {
-      page_name: 'about',
-      page_data: '{
-          "Purpose" : "The purpose of this organization shall be to advocate for maternal and child public health-related issues for all students.",
+# if Page.count < 3
```
Comment (jswood23) [inline db/seeds.rb]: Why are we commenting this out again? We need this code so that there will only be one of each page in the "Manage Pages" page.

**Review by jswood23 — CHANGES_REQUESTED** [review]
The css changes look good but I have a question about the seed file change.

File: db/seeds.rb (hunk 1 above)
Comment (jswood23) [reply db/seeds.rb]: I would change it to `if Page.all.count.zero?` or `if Page.all.count < 3`

Comment (NR229) [reply db/seeds.rb]: I changed it back, not sure why it changed in the first place.

Comment (NR229) [reply db/seeds.rb]: Also, I saw your text; I think it looks good without the border line above the logos

**Review by jswood23 — APPROVED** [review]
Looks good

**Review by ynzhang09 — APPROVED** [review]
Code and deployment looks good

**Pull request merged**